examples/demos/customView.js

```diff
@@ -44,18 +51,23 @@ MyWeek.navigate = (date, action) => {
 }
 
 MyWeek.title = (date, { formats, culture }) => {
-  return `My awesome week: ${Date.toLocaleString()}`
+  return `My awesome week: ${date.toLocaleString()}`
 }
 
-let CustomView = props => (
-  <div>
-    <BigCalendar
-      events={events}
-      defaultDate={new Date(2015, 3, 1)}
-      views={{ month: true, week: MyWeek }}
-      test="io"
-    />
-  </div>
-)
-
-export default CustomView
+
+let CustomView = React.createClass({
```

Comment:
Could you update this to use es6 instead?

Reply:
yeah, no problem